src/tdvp.jl

```diff
@@ -84,11 +84,12 @@ function tdvp(PH,
     kwargs, :write_when_maxdim_exceeds, nothing
   )
 
-  # exponentiate kwargs
-  exponentiate_tol::Float64 = get(kwargs, :exponentiate_tol, 1e-14)
-  exponentiate_krylovdim::Int = get(kwargs, :exponentiate_krylovdim, 20)
-  exponentiate_maxiter::Int = get(kwargs, :exponentiate_maxiter, 1)
-  exponentiate_verbosity::Int = get(kwargs, :exponentiate_verbosity, 0)
+  # exponentiate keyword args
+  exponentiate_kw = (;ishermitian= get(kwargs, :exponentiate_ishermitian, true),
+                      tol= get(kwargs, :exponentiate_tol, 1e-14),
+                      krylovdim= get(kwargs, :exponentiate_krylovdim, 20),
+                      maxiter= get(kwargs, :exponentiate_maxiter, 1),
```

Comment:
I agree, the maxiter default should be set to a larger number. If maxiter is hit, the code needs to raise an error and abandon the propagation. Setting maxiter to a finite value just helps to avoid being stuck in exponentiate forever.

Reply:
This is great feedback. I can just set the defaults to the ones mentioned by KrylovKit, how's that sound? Or is 100 too much for maxiter in your experience, Benedikt? 

Two other things that could then be added:

- adding another keyword that plays the role of epsilon in that expression epsilon/abs(t) above for a way of setting `tol`
- code which checks the return value info of `exponentiate` to check things like if maxiter was hit and then throw an error

Reply:
It seems like there should be some theory also about how to set that epsilon (as in tol=epsilon/abs(t)) in a way that is related to the cutoff used in the two-site algorithm. But that's probably getting to fancy for now, and anyway isn't relevant in the one-site algorithm.